tests/infra/suspension.py

```diff
@@ -42,10 +42,10 @@ def update_view_info(network, view_info):
 def get_node_local_commit(node):
     with node.node_client() as c:
         r = c.get("commit")
-        return r.seqno, r.global_commit
```

Comment:
global_commit is still there, and was never tied to the transaction (it's just whatever the consensus thinks it is right now). Its removal is a separate matter.